### ISSUE TITLE
RedditAccount contains incorrect values after call to meJson()

### ISSUE THREAD

**Issue by JamesGold23** [corydissinger/raw4j#16]
I used my own account to test this method. The RedditAccount:

RedditAccount [commentKarma=0, created=1316419867, createdUtc=0, hasMail=null, hasModMail=null, id=5vx5s, linkKarma=0, name=JamesGold, over18=false]

Several of these values are incorrect. However, the HTTP response body contained the correct values, so something is going wrong with the JSON parsing of the body.

**Comment by corydissinger** [comment]
Good catch. Please pull down the latest commit and verify the values are now being set.

The issue was indeed in the JSON parsing, I failed to notice that the JSON response had _ (underscores) and needed to use Jackson's `@JsonProperty` annotation to map them.

After you get back to me I'll close the issue.
